geode-connectors/src/main/java/org/apache/geode/connectors/jdbc/internal/cli/CreateMappingCommand.java

```diff
@@ -97,5 +98,6 @@ public void updateClusterConfig(String group, CacheConfig cacheConfig, Object el
         .stream()
         .filter(regionConfig -> regionConfig.getName().equals(newCacheElement.getRegionName()))
         .forEach(regionConfig -> regionConfig.getCustomRegionElements().add(newCacheElement));
+    return true;
```

Comment:
where is the return value used?

Reply:
The return value is used in  `CommandExecutor.invokeCommand`. Individual command classes that override `updateClusterConfig` will return `true` or `false` to indicate whether the configuration was updated or not (or possibly throw an exception if an error occurred updating the configuration).